pkgpanda/__init__.py

```diff
@@ -442,7 +447,14 @@ def validate_username(username):
         if not re.match(username_regex, username):
             raise ValidationError("Username must begin with `dcos_` and only have a-z and underscore after that")
 
-    def add_user(self, username):
+    @staticmethod
+    def validate_group(group):
+        try:
+            grp.getgrnam(group)
```

Comment:
None is allowed as a value for group below in add_user, should deal with that here as well probably.